utils/cliutils/commandsflags.go

```diff
@@ -1007,9 +1007,9 @@ var flagsMap = map[string]cli.Flag{
 		Name:  usesPlugin,
 		Usage: "[Default: false] Set to true if the Gradle Artifactory Plugin is already applied in the build script.` `",
 	},
-	UseWrapper: cli.BoolFlag{
+	UseWrapper: cli.BoolTFlag{
```

Comment:
Since we don't want to change the default value of the `--user-wrapper` option of the `gradlec` command, I suggest splitting this option into two different options -
`AuditUseWrapper` and `ConfigUseWrapper`.